Libplanet.Tests/Action/ActionEvaluatorTest.cs

```diff
@@ -581,7 +583,7 @@ public void EvaluateTx()
             var actionEvaluator = new ActionEvaluator<DumbAction>(
                 policyBlockAction: null,
                 blockChainStates: NullChainStates<DumbAction>.Instance,
-                trieGetter: null);
+                trieGetter: null, genesisHash: null);
```

Comment:
For consistency, please give a line to each argument.

```suggestion
                trieGetter: null,
                genesisHash: null);
```

Reply:
Sure! I will look into it again